build.gradle

```diff
@@ -98,15 +98,16 @@ tasks.withType(Sign).configureEach {
     notCompatibleWithConfigurationCache("$name task does not support configuration caching")
 }
 
-def gitHubReleaseName() {
-    return version.toString()
-}
-
 def gitReleaseTag() {
-    return "v${version}"
+    return version.map { "v$it" }
 }
```

Comment:
I feel like `gitReleaseTag` could be removed and the contents inlined. It's only called in a single place and feels redundant.

Reply:
Agreed. I'll make that change as I go through the rest of the repos too - they follow this same pattern.

Reply:
Actually it is called twice, if that changes your opinion.

Reply:
Personally I kinda like keeping it in that case.

Reply:
See comment above.

Reply:
Resolving given it has been inlined.